src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -43,6 +43,7 @@
 # onefuzztypes
 class Proxy(ORMMixin):
     timestamp: Optional[datetime.datetime] = Field(alias="Timestamp")
+    created_timestamp = Optional[datatime.datetime] = Field(alias="Created Timestamp")
```

Comment:
Suggestion, use `default_factory` to create this.

`Field(default_factory=datetime.datetime.utcnow())`